Code/GraphMol/FileParsers/MaeWriter.cpp

```diff
@@ -230,7 +224,7 @@ int bondTypeToOrder(const Bond &bond) {
     case Bond::BondType::DATIVE:
       return 0;
     default:
-      throw UnsupportedBondException(
+      throw FileParseException(
```

Comment:
I think this exception (and the other one below) would be better as a `ValueErrorException`. It's not really a parse exception.
I see that there are a couple of other exiisting uses of `FileParseException` in `MaeWriter.cpp`, but we don't need to add more.

Reply:
Sounds good. I wasn't quite convinced about `FileParseException` either.